saleor/payment/gateways/adyen/utils/common.py

```diff
@@ -356,7 +356,12 @@ def request_data_for_gateway_config(
         country_code = country.code
     else:
         country_code = Country(settings.DEFAULT_COUNTRY).code
-    channel = get_checkout_metadata(checkout).get_value_from_metadata("channel", "web")
+    checkout_metadata = get_checkout_metadata(checkout)
+    if checkout_metadata:
+        channel = checkout_metadata.get_value_from_metadata("channel", "web")
+    else:
+        channel = "web"
```

Comment:
I know it's not in the scope of that pr, but what is this "web" value?
I see we have more occurrences of that string, maybe it's worth to extract it to some variable?

Reply:
@kadewu  It is a value sent to Adyen. It is used to determine the channel related to the processed payment. At least this is what I remember 😅 
Honestly, there is no reason to invest time in AdyenPlugin right now, as we are moving from plugins to apps.